app/authentication/server/startup/index.js

```diff
@@ -363,8 +363,17 @@ Accounts.validateNewUser(function(user) {
 		return true;
 	}
 
-	if (settings.get('Accounts_Registration_AuthenticationServices_Enabled') === false && settings.get('LDAP_Enable') === false && !(user.services && user.services.password)) {
-		throw new Meteor.Error('registration-disabled-authentication-services', 'User registration is disabled for authentication services');
+	if (settings.get('Accounts_Registration_AuthenticationServices_Enabled') === false) {
+		if (settings.get('LDAP_Enable') === false && !(user.services && user.services.password)) {
+			throw new Meteor.Error('registration-disabled-authentication-services', 'User registration is disabled for authentication services');
+		} else if (settings.get(/^Accounts_OAuth_Custom-[a-z0-9_]+$/i).length > 0) {
+			const customOAuthServicesInUse = settings.get(/^Accounts_OAuth_Custom-[a-z0-9_]+$/i)
```

Comment:
The I check if any custom OAuth services were set by checking `settings.get(/^Accounts_OAuth_Custom-[a-z0-9_]+$/i).length > 0`.

If the custom OAuth services were set as well as the user login with one of it, the `customOAuthServicesInUse.length > 0` reaches and rejects the user creation